README.md

```diff
@@ -125,12 +125,12 @@ Sets the protocol to be included in the exception rule, either: 'TCP' or 'UDP'.
 ##### `local_port`
 
 Defines the local port to be included in the exception for port-based exception
-rules, either: an integer between 1 and 65535 or the string 'any'.
+rules, either: an integer between 1 and 65535, a port range (two integers separated by a hyphen, a comma separated list of integers, or the string 'any'.
```

Comment:
Would it be difficult to make it accept an array of integers (instead of comma separated), and have the module do the right thing?

Reply:
I'd have to think on that but I imagine it would require a lambda and maybe if statements before the powershell portion. The way it is currently, Windows server expects "individual numbers from 0 through 65535, a range, such as 5000-5020, or a comma-separated list of numbers and ranges". So the powershell command can run with the given input without manipulation. If you had an array you'd have to add the commas back in.